### PR TITLE
Merge master

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

**Comment by gpolitis** [comment]
why do we need the new dependency? (jitsi-xmpp-extensions)

**Comment by bgrozev** [comment]
Ops, it's already there. I removed it.

File: src/main/java/org/jitsi/videobridge/cc/AdaptiveTrackProjection.java [hunk 1]
```diff
@@ -215,6 +215,19 @@ public boolean accept(@NotNull VideoRtpPacket videoRtpPacket)
             logger.warn(
                 "Dropping an RTP packet, no quality index (SSRC=" +
                     videoRtpPacket.getSsrc() + ").");
+            MediaStreamTrackDesc sourceTrack = getSource();
+            if (sourceTrack != null)
+            {
+                logger.warn(
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/cc/AdaptiveTrackProjection.java]: confused by this, if we find the source track is not null we drop it?  and if it is null we  drop it?

Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/cc/AdaptiveTrackProjection.java]: oh, ok...after expanding i think i see now. nevermind.

**Pull request merged**